python/lsst/ap/association/transformDiaSourceCatalog.py

```diff
@@ -187,7 +188,7 @@ def run(self,
             Filter band of the science image.
         ccdVisitId : `int`
             Identifier for this detector+visit.
-        funcs : `lsst.pipe.tasks.functors.Functors`
+        funcs : `lsst.pipe.tasks.functors.Functors`, optional
```

Comment:
I think we can just remove this `funcs` entirely: it's entirely unused: the code uses `self.funcs`, set in `__init__`.